lib/linguist/languages.yml

```diff
@@ -6285,6 +6285,15 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+ShockScript:
+  type: programming
+  color: "#ff9070"
+  extensions:
+  - '.sx'
+  - '.shockscript'
+  tm_scope: source.sx
+  ace_mode: actionscript
+  language_id: 850000000
```

Comment:
Please follow the guidelines to add a new language. This id shouldn't be set manually.